packages/babel-compiler/babel-compiler.js

```diff
@@ -79,7 +79,8 @@ BCp.processOneFileForTarget = function (inputFile, source) {
       extraFeatures.nodeMajorVersion = parseInt(process.versions.node);
     }
 
-    if (arch !== "web.browser.legacy") {
+    if (arch !== "web.browser.legacy" &&
+        ! hasOwn.call(extraFeatures, "runtime")) {
       extraFeatures.runtime = {
```

Comment:
Specifically, [this line](https://github.com/meteor/meteor/blob/a5590259288083f6dc4dec4d42cca3dfb62af4fc/packages/non-core/coffeescript-compiler/coffeescript-compiler.js#L25) from `coffeescript-compiler` was being overridden here, before this change.